enterprise/backend/test/metabase_enterprise/advanced_permissions/common_test.clj

```diff
@@ -858,29 +858,24 @@
   (mt/test-drivers (mt/normal-drivers-with-feature :uploads :schemas)
     (testing "GET /api/database and GET /api/database/:id responses should include can_upload depending on unrestricted data access to the upload schema"
       (mt/with-model-cleanup [:model/Table]
-        (let [schema-name (sql.tx/session-schema driver/*driver*)]
+        (let [schema-name (sql.tx/session-schema driver/*driver*)
+              db-id       (u/the-id (mt/db))]
           (upload-test/with-upload-table! [table (upload-test/create-upload-table! :schema-name schema-name)]
-            (let [db-id (u/the-id (mt/db))]
-              (mt/with-temp [:model/Table {} {:db_id db-id :schema "some_schema"}]
-                (mt/with-temporary-setting-values [uploads-enabled      true
-                                                   uploads-database-id  db-id
-                                                   uploads-schema-name  schema-name
-                                                   uploads-table-prefix "uploaded_magic_"]
```

Comment:
This test boilerplate is taken care of by `upload-test/with-upload-table!` now. 

https://github.com/metabase/metabase/blob/595166f34716ab4d820cb655b7ae0f89d0302af8/test/metabase/upload_test.clj#L322-L323